common/lib/xmodule/xmodule/partitions/partitions_service.py

```diff
@@ -166,6 +167,9 @@ def get_group(self, user, user_partition, assign=True):
         If the user has not yet been assigned, a group will be chosen for them based upon
         the partition's scheme.
         """
+        if not user_partition:
+            return None
```

Comment:
Same thing as my `course` comment above. Do we anticipate this being called with an incorrect `user_partition`? Just feels odd to me because the code has been running without this check and I don't believe we have been seeing errors because of it.